WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -57,9 +60,11 @@ struct HubMenu: View {
     private struct TopBar: View {
         let storeTitle: String
         let storeURL: String?
+        var presenSwitchStore: (() -> Void)?
```

Comment:
Nit: typo `presenSwitchStore` 😅 Since this property is a closure, how about naming it with a noun instead?
```suggestion
        var switchStoreHandler: (() -> Void)?
```

Reply:
Good catch, fixed 👍